examples/console.rs

```diff
@@ -127,10 +128,22 @@ impl DrawingBackend for TextDrawingBackend {
     fn draw_text(
         &mut self,
         text: &str,
-        _style: &TextStyle,
+        style: &TextStyle,
         pos: (i32, i32),
     ) -> Result<(), DrawingErrorKind<Self::ErrorType>> {
-        let offset = pos.1.max(0) * 100 + pos.0.max(0);
+        let (width, height) = self.estimate_text_size(text, &style.font)?;
+        let (width, height) = (width as i32, height as i32);
+        let dx = match style.pos.h_pos {
+            HPos::Left => 0,
+            HPos::Right => -width,
+            HPos::Center => -width / 2,
+        };
+        let dy = match style.pos.v_pos {
+            VPos::Top => 0,
+            VPos::Center => -height / 2,
+            VPos::Bottom => -height,
```

Comment:
I went through the code, it seems the new code doesn't really guarantee the text isn't in the bouncing box.  Since the decedent part will be always out of box, and this should be avoid.